scripts/pre-commit/validate_migration_scripts.py

```diff
@@ -68,10 +72,10 @@ def check_new_migration_file_format(new_migration_files: tp.List) -> None:
     for file in new_migration_files:
         if file == BOOTSTRAP_FILE:
             continue
-        if not re.match(r"^V\d{4}.\d{2}.\d{2}.\d{1,2}__.+\.sql$", file):
+        if not re.match(r"^V\d{4}.\d{2}.\d{2}.\d{1}__.+\.sql$", file):
```

Comment:
I think this regex isn't correct. Can you reconfirm? 
We need to escape the `.` to literally match with a dot in the file name, if not it will match any char. 

You can also omit the `{1}` from last digit matching. `\d`  will match only single digit
```
r"^V\d{4}\.\d{2}\.\d{2}\.\d__.+\.sql$"
```